modules/ROOT/pages/_partials/commons/common-p2psync-websocket.adoc

```diff
@@ -214,30 +216,30 @@ include::{root-partials}p2p-api.adoc[leveloffset=+1]
 === Authentication
 // include::{root-partials}p2p-security.adoc[leveloffset=+1]
 
-Peer-to-peer sync supports <<using-basic-authentication, Basic Authentication>> and <<using-tls,TLS Authentication>>.
-For anything other than test deployments, we strongly encourage the use of TLS. In fact, peer-to-peer sync using URLEndpointListener is encrypted using TLS by default.
+Peer-to-Peer sync supports <<using-basic-authentication, Basic Authentication>> and <<using-tls,TLS Authentication>>.
+For anything other than test deployments, we strongly encourage the use of TLS. In fact, Peer-to-Peer sync using URLEndpointListener is encrypted using TLS by default.
 
-The authentication mechanism used is defined at endpoint level, meaning that it is independent of the database being replicated.
-So when replicating multiple database instances you may use basic on one instance an another authentication method on other instances.
+The authentication mechanism is defined at the endpoint level, meaning that it is independent of the replicated database.
+For example, you may use basic authentication on one instance and TLS authentication on another when replicating multiple database instances.
 
 NOTE: The Minimum supported version of TLS is TLS 1.2.
 
-Peer-to-Peer Synchronization using URLEndpointListener supports certificate based authentication of the server and-or listener:
+Peer-to-Peer synchronization using URLEndpointListener supports certificate based authentication of the server and-or Listener:
 
 * Replicator certificates can be: self signed, from trusted CA or anonymous (system generated).
 * Listeners certificates may be: self signed or trusted CA signed.
 +
-Where a TLS certificate is not explicitly specified for the listener, the Listener implementation will generate anonymous certificate to use for encryption
+Where a TLS certificate is not explicitly specified for the Listener, the Listener implementation will generate anonymous certificate to use for encryption
 
 * The URLEndpointListener supports the ability to opt out of TLS encryption communication.
 +
-Active clients replicating with a URLEndpointListener have the option to skip validation of server certificates when the listener is configured with self-signed certificates.
+Active clients replicating with a URLEndpointListener have the option to skip validation of server certificates when the Listener is configured with self-signed certificates.
 +
 This option is ignored when dealing with CA certificates.
 
 [#using-secure-storage]
 === Using Secure Storage
-The use of TLS, its associated keys and certificates requires using secure storage to minimize the chances of a security breach.
+TLS and its associated keys and certificates require using secure storage to minimize the chances of a security breach.
```

Comment:
Hmm, not sure if "the use of" might actually be warranted here...
The TLS spec itself doesn't require this, it's more a best practice.
@dinaabennett what do you think?